src/schema/objects/common_principles.yaml

```
@@ -97,6 +100,9 @@ modality:
     the technique is sufficiently uniform to define the modalities `eeg`, `meg` and `ieeg`.
     When applicable, the modality is indicated in the **suffix**.
     The modality may overlap with, but should not be confused with the **data type**.
+raw:
+  display_name: raw dataset
+  description: A raw BIDS dataset is data that have been curated into BIDS from a non-BIDS source.
```

Looking at other definitions, they are continuation of the phrase "{display_name} is ..." and hence I would adjust here too to

```suggestion
  description: A BIDS dataset that have been curated into BIDS from a non-BIDS source(s), for example data from acquisition hardware.
```